3d_segmentation/swin_unetr_brats21_segmentation_3d.ipynb

```diff
@@ -0,0 +1,1021 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !pip install git+https://github.com/Project-MONAI/MONAI#egg.gitmonai@0.8.1+271.g07de215c 
</pre><p>Could you please help change to install some MONAI official version?</p><p> </p><p>Thanks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/750/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Sure. Which official MONAI version has Swin UNETR ? Can adopt it.